.github/workflows/cargo_fmt.yml

```diff
@@ -34,41 +38,54 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check
+          args: -- --check
 
-      - name: Cargo format fuzz/
+      - name: Cargo format libraries/opensk
         uses: actions-rs/cargo@v1
         with:
+          toolchain: nightly
           command: fmt
-          args: --manifest-path fuzz/Cargo.toml --all -- --check
+          args: --manifest-path libraries/opensk/Cargo.toml --all -- --check
```

Comment:
nit: We should also be able to get rid of the `--all` here

Same throughout this file.

Reply:
Done